skbio/metadata/intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
I don't know anything about `rseqc` but it appears to have copied all of `bx-python` without attribution. 

This code is indeed from `bx-python` which is MIT licensed and I have no objection to including it here. In fact, I'd be happy to see anything people find useful from `bx-python` moved into `scikit-bio`. 

ping @brentp @ianschenck